code/modules/wiremod/components/action/speech.dm

```diff
@@ -35,6 +35,8 @@
 		// Prevents appear as the individual component if there is a shell.
 		if(shell)
 			shell.say(message.input_value)
+			shell.log_talk(message.input_value, LOG_SAY, tag="circuit speech | Built by [parent.get_creator()].")
 		else
 			say(message.input_value)
+			log_talk(message.input_value, LOG_SAY, tag="circuit speech | Built by [parent.get_creator()].")
```

Comment:
These messages will be a bit misleading because it's in this format:
`shell_object (info on creator)`